examples/blender/blender.py

```diff
@@ -1,11 +1,15 @@
+#!/usr/bin/env python3
+
```

Comment:
Not needed in my opinion - but I'm not sure. Can someone else comment on that?

Reply:
Without hasbang you won't be able to run it directly.

Reply:
@jiivan I guess, @filipgolem 's argument was that you can run it with `python3 blender.py` ...

Reply:
But why disable the `./blender.py` way?

Reply:
hashbang is useful yen harmless. Good change imo